base/reflection.jl

```diff
@@ -809,10 +809,10 @@ function _dump_function_linfo(linfo::Core.MethodInstance, world::UInt, native::B
 end
 
 """
-    code_llvm([io], f, types)
+    code_llvm(io=STDOUT, f, types)
```

Comment:
See #4902, but before a decision has been made there I think this should either be
```julia
    code_llvm([io=STDOUT,] f, types)
```
or
```julia
    code_llvm(io, f, types)
    code_llvm(f, types)
```

Reply:
You are right, i didn't remember this was still undecided. I do like your first suggestion better, as it shows the default value. 
Going to switch to that version